tests/test_trackingstore.py

```diff
@@ -379,6 +381,49 @@ def test_get_run_client_error(mocker):
         store.get_run(RUN_UUID_HEX_STR)
 
 
+def test_update_run_info(mocker):
+    faculty_run = mocker.Mock()
+    mock_client = mocker.Mock()
+    mock_client.update_run_info.return_value = faculty_run
```

Comment:
The above is incorrect with the current implementation of `update_run_info` in the client. It returns None.